elementary/cli/cli.py

```diff
@@ -71,5 +71,15 @@ def invoke(self, ctx: click.Context) -> Any:
     help="Open source data reliability solution (https://docs.elementary-data.com/)"
 )
 
+
+@click.command(cls=ElementaryCLI)
+@click.version_option(
+    version=package.get_package_version(),
+    message="Elementary installed version %(version)s.",
+)
+def cli():
```

Comment:
Once you define `cli` here, you override the cli definition on line 70.
To support the same implementation of help that we define on line 70:
1. delete the `cli` definition on line 70
2. Edit `@click.command(cls=ElementaryCLI)` on line 75 to be:
```python
@click.command(
    cls=ElementaryCLI,
    help="Open source data reliability solution (https://docs.elementary-data.com/)",
)
```

That way we will have the version and the wanted help message 🙂

Reply:
Thanks @IDoneShaveIt for poiniting this out. I will correct it next commit. Also do you have any feedback on the statement "Elementary installed version %(version)s.". I mean any other sentence or keyword you suugest to display?

Reply:
You can write "Elementary version %(version)s." 🙂

Reply:
Fine!